src/main/java/core/basesyntax/suppliers/FigureSupplier.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.suppliers;
+
+import core.basesyntax.Figure;
+import core.basesyntax.figure.Circle;
+import core.basesyntax.figure.IsoscelesTrapezoid;
+import core.basesyntax.figure.Rectangle;
+import core.basesyntax.figure.RightTriangle;
+import core.basesyntax.figure.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public double getRandomlength() {
+        return random.nextDouble();
+    }
+
+    public Figure getRandomFigure() {
+        int indexOfFigure = random.nextInt(5);
```

Comment:
5 - should  be constant 
```suggestion
        int figureIndex = random.nextInt(5);
```